arch/arm64/configs/sun50iw1p1smp_linux_defconfig

```diff
@@ -161,7 +161,7 @@ CONFIG_TIMERFD=y
 CONFIG_EVENTFD=y
 CONFIG_SHMEM=y
 CONFIG_AIO=y
-CONFIG_EMBEDDED=y
+# CONFIG_EMBEDDED is not set
```

Comment:
in order to disable DEBUG build of kernel "EMBEDDED" needs to be disabled (please refer to help in kernel parameter)